source/includes/_companies.md

```diff
@@ -410,6 +410,7 @@ curl --location --request PUT 'https://wellthiedemo-smallbusiness-qa.affordablec
     "zip_code_input": "10001",
     "contact_name": "Test",
     "contact_email": "test@test.com",
+    "effective_date": "1/1/1970",
```

Comment:
this may be wrong, i just put something down since it was missing from the curl call